orchestration/dagster_orchestration/hca_manage/main.py

```diff
@@ -43,18 +42,35 @@ def run(arguments=None):
                         required=True)
     subparsers = parser.add_subparsers(dest='command')
 
+    # validation checks
     parser_check = subparsers.add_parser("check",
                                          help="Command to check HCA datasets for duplicates and null file references")
     # only allow if env is prod
     parser_check.add_argument("-p", "--project", help="The Jade project to target, defaults to correct project for dev")
     parser_check.add_argument("-d", "--dataset", help="The Jade dataset to target", required=True)
     parser_check.add_argument("-r", "--remove",
-                              help="Remove problematic rows. If flag not set, will only check for presence of problematic rows.",
+                              help="Remove problematic rows. If flag not set, will only check for presence of problematic rows",
                               action="store_true")
 
-    parser_snapshot = subparsers.add_parser("snapshot", help="Command to create a snapshot for an HCA release")
-    parser_snapshot.add_argument("-d", "--dataset", help="The Jade dataset to target", required=True)
-    parser_snapshot.add_argument("-q", "--qualifier", help="Optional qualifier to append to the snapshot name")
+    # snapshot management
+    parser_snapshot = subparsers.add_parser("snapshot", help="Command to manage snapshots")
```

Comment:
Out of scope for now but: we should consider refactoring this method, it's getting quite large.